### ISSUE TITLE
Android warns about unchecked or unsafe operations

### ISSUE THREAD

**Issue by hgl** [Sh1d0w/multi_image_picker#101]
**Bug Report**

`multi_image_picker` version: 4.0.0

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel dev, v1.4.4, on Mac OS X 10.13.6 17G5019, locale en-CN)
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
[✓] Android Studio (version 3.2)
[✓] VS Code (version 1.32.3)
[✓] Connected device (3 available)
```

When building flutter android, it warns about about unchecked or unsafe operations. Although the apk can run without issues.

I wonder if it's intended?

```
Launching lib/main.dart on Android SDK built for x86 64 in debug mode...
Note: /path/to/flutter_sdk/.pub-cache/hosted/pub.dartlang.org/multi_image_picker-4.0.0/android/src/main/java/com/vitanov/multiimagepicker/MultiImagePickerPlugin.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
Built build/app/outputs/apk/debug/app-debug.apk.
```

**Comment by Sh1d0w** [comment]
This is now fixed in version 4.0.1. Thanks for the report.

**Comment by hgl** [comment]
Thanks for the quick fix.
